SpatialGDK/Source/SpatialGDK/Public/Interop/ReserveEntityIdsHandler.h

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "Interop/SpatialOSDispatcherInterface.h"
+#include "SpatialCommonTypes.h"
+
+#include <WorkerSDK/improbable/c_schema.h>
+#include <WorkerSDK/improbable/c_worker.h>
+
+#include "Connection/SpatialOSWorkerInterface.h"
+
+DECLARE_CYCLE_STAT(TEXT("ReserveEntityHandler"), STAT_ReserveEntityHandler, STATGROUP_SpatialNet);
+
+namespace SpatialGDK
+{
+class ReserveEntityHandler
```

Comment:
ReserveEntityIdsHandler

Reply:
I'm not sure this object has much value living separately to the EntityPool itself. Is there a reason you split it?

Reply:
I see it matches the pattern of the other handlers though - so I understand it's usefulness more. Can disregard my comment.

Reply:
Oh and it's also used in the SnapshotManager - gotcha